src/inc/cordebug.idl

```diff
@@ -810,6 +810,21 @@ interface ICorDebugDataTarget3 : IUnknown
                              length_is(*pcFetchedModules)] ICorDebugLoadedModule *pLoadedModules[]);
 };
 
+
+[
+    object,
+    uuid(E799DC06-E099-4713-BDD9-906D3CC02CF2),
+    local,
+    pointer_default(unique)
+]
+interface ICorDebugDataTarget4 : IUnknown
+{
+    /*
+    * gives back a process id
+    */
```

Comment:
Can you add to this comment that the process could still be remote or a dump, and thus there is no guarantee that trying to use OS APIs against this PID will be fruitful.